codespell_lib/data/dictionary.txt

```diff
@@ -40527,6 +40528,7 @@ pincher->pinscher
 pinnaple->pineapple
 pinockle->pinochle
 pinoneered->pioneered
+pint->point
```

Comment:
[`pint`](https://www.oed.com/search/dictionary/?q=pint) is a word and cannot be added to the typos dictionary.

Reply:
true, thanks

Reply:
This commit has been dropeed